vortex-serde/src/layouts/write/writer.rs

```diff
@@ -253,13 +230,31 @@ impl <T> ColumnChunkAccumulator for TypedColumnChunkAccumulator<T> {
     }
 
     fn push_stat(&mut self, stat: Stat, value: Option<Scalar>) -> VortexResult<()> {
```

Comment:
This is a bit of a weird API that you push one stat at a time into vecs. It makes it too easy to have unaligned vectors no?

Can you take a struct of optional stats that gets pushed onto all vectors at once?